apps/frontend/src/pages/ConfirmEmail.tsx

```diff
@@ -10,7 +10,7 @@ const ConfirmEmail = () => {
   useEffect(() => {
     const confirmEmail = async () => {
       try {
-        await axios.post("/user/confirm-email", { code: token });
+        await axios.post("/api/user/confirm-email", { code: token });
```

Comment:
If you also make sure the token is set as a cookie, then none of this auth implementation is required